docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -0,0 +1,285 @@
+    1. Click **Add CI Job **next to the name of the project to add.
+    2. Optional: If you do not see the repository you want to add, follow the succeeding guide on **detecting GitHub repositories**.
+    3. Click **Commit file**. This commits a `semgrep.yml` file containing pertinent scan parameters, such as schedule, what branch to scan on, and so on.
+    4. Semgrep App generates a token composed of a **Secret name** and **Secret value**. Copy and paste these into your repository's settings. Links are provided by Semgrep App.
+    5. Click **The secret's there, continue**.
+    6. Semgrep App creates a file for the CI job. Select toggles for desired features, such as scanning on a schedule.
+6. For **GitLab CI/CD**:
+    1. Create a Semgrep App token by clicking **Settings > Tokens > Create new token**, then return to the project setup by clicking on the back arrow on your browser.
+    2. Add the Semgrep App token as a secret [CI/CD variable](https://docs.gitlab.com/ee/ci/variables/#custom-cicd-variables) named `SEMGREP_APP_TOKEN`.
+    3. Select toggles to determine scan behavior.
+    4. Copy the snippet provided to your `.gitlab-ci.yml` file and commit it.
+7. For **CircleCI**:
+    1. Create a Semgrep App token by clicking **Settings > Tokens > Create new token**, then return to the project setup by clicking on the back arrow on your browser.
+    2. Add the Semgrep App token as a project [environment variable](https://circleci.com/docs/2.0/env-vars/) named `SEMGREP_APP_TOKEN`.
+    3. Copy the snippet provided and commit the `circleci/config.yml` file.
+8. If successful, Semgrep App scans the repository for the first time using default, pre-selected rules.
+
+To ensure that your GitHub repository is **detected** by Semgrep App:
+
+1. Log into GitHub.
+2. Click your **profile photo > Settings > Applications**.
+3. On the `semgrep-app` entry, click **Configure**.
+4. Under Repository access select an option to provide access:
+    1. All repositories will display all current and future public and private repositories.
+    2. Only select repositories will display explicitly selected repositories.
+
+
+### Running a scan
+
+By default, scans are triggered through the following parameters, which are defined during a project's initial setup in Semgrep App:
+
+
+* Either a daily or weekly schedule.
+* Upon every PR or MR.
+* Upon every update to the `semgrep.yml` file.
+
+To change these scan parameters, either:
+
+* Edit the `semgrep.yml` file manually.
+
+* Remove the project and redo the steps described in Adding a project section.
+
+Additional scan parameters include:
+
+**Rule recommendation**
+    Select this toggle to receive rule recommendations in the Rule Board based on the framework and language of the repository.
+
+**Autofix**
+    Select this toggle to enable autofix, which creates suggestions in addition to PR or MR comments. For example, a rule may suggest using a function such as `logging.debug()` instead of `print()`.

```

Comment:
```suggestion
    <dd>Select this toggle to enable autofix, which creates suggestions in addition to PR or MR comments. For example, a rule may suggest using a function such as <code>logging.debug()</code> instead of <code>print()</code>.</dd>
```

Reply:
I hope that the `<code>something</code>` will be rendered the same as with markdown ``...

Reply:
@adamkvitek I'll test